src/templates/it/base.html

```diff
@@ -0,0 +1,172 @@
+{% extends "base/base.html" %}
+{% block description %}Web Almanac è un report annuale sullo stato del Web che combina l'esperienza della comunità Web con i dati e le tendenze dell'archivio HTTP.{% endblock %}
+
+{% block twitter_image_alt %}{{ year }} Web Almanac{% endblock %}
+{% block skip_navigation %}Salta la navigazione{% endblock %}
+{% block organization %}Web Almanac di HTTP Archive{% endblock %}
+{% block web_almanac_logo %}
+  <span class="wa">Web Almanac</span>
+  <span class="line-group">
+    <span class="pre">di</span>
+    <span class="ha">HTTP Archive</span>
+  </span>
+{% endblock %}
+
+{% block intro_title %}Web Almanac{% endblock %}
+{% block intro_sub_title %}Rapporto annuale<br> <b>sullo stato del web</b> di HTTP Archive{% endblock %}
+
+{% block mission %}
+<p>
+La nostra missione è combinare le statistiche grezze e le tendenze del HTTP Archive con l'esperienza della comunità web. Il Web Almanac è un rapporto completo sullo stato del web, supportato da dati reali e da esperti web di fiducia. È composto da {{supported_chapters | length}} capitoli che abbracciano aspetti del page content, user experience, publishing e distribution.
+</p>
+{% endblock %}
+{% block read_last_years_almanac %}Leggi il Web Almanac del {{ year | int - 1 }}{% endblock %}
+
+{% block http_archive_link %}Pagina iniziale HTTP Archive{% endblock %}
+
+{% block page_navigation %}Navigazione tra le pagine{% endblock %}
+
+{% block help_translate %}Help Translate{% endblock %}
+{% block translation_not_available %}Inglese - non disponibile in inglese{% endblock %}
+{% block language_switcher %}Seleziona la lingua{% endblock %}
+{% block year_switcher %}Seleziona l'anno{% endblock %}
+{% block table_of_contents_switcher %}Seleziona sommario{% endblock %}
+
+{% block home %}Pagina iniziale{% endblock %}
+{% block table_of_contents_title %}Sommario{% endblock %}
+{% block contributors_title %}Contributori{% endblock %}
```

Comment:
Should we put "Curatori" or "A cura di" ? 

@chefleo  @Zizzamia

Reply:
"Collaboratori"?

Reply:
"Collaboratori" sounds better cc @chefleo

Reply:
Adding as suggestion to allow it to be committed easily:

```suggestion
{% block contributors_title %} Collaboratori{% endblock %}
```